app/models/miq_group.rb

```diff
@@ -124,7 +128,7 @@ def self.get_httpd_groups_by_user(user)
     rescue => err
       raise _("Unable to get groups for user %{user_name} - %{error}") % {:user_name => username, :error => err}
     end
-    user_groups.first
+    strip_group_domains(user_groups.first)
```

Comment:
@jvlcek Will `user_groups.first` return on object on which `collect` can be called?